src/methods/centroid.jl

```diff
@@ -169,7 +169,7 @@ end
 # It combines two (point, area) tuples into one, taking
 # the average of the centroid points weighted by the
 # area of the geom they are from.
-function _combine_centroid_and_area(((x1, y1), area1), ((x2, y2), area2))
+function _combine_centroid_and_area(((x1, y1), area1)#=::Tuple{Tuple{<: Real, <: Real}, <: Real}=#, ((x2, y2), area2)#=::Tuple{Tuple{<: Real, <: Real}, <: Real}=#) # ::Tuple{Tuple{<: Real, <: Real}, <: Real}
```

Comment:
Is there a reason we can't put these in the comments above? Is this for documentation generation purposes? It would be easier to read above in my opinion. Also, can we make the comments above use multi-line comments? I have been doing that, again just for readability.

Reply:
I think I was trying to remove a type instability and wrote this - thanks for catching it!  Will remove.